grakn-engine/src/test/java/ai/grakn/engine/EngineTestHelper.java

```diff
@@ -0,0 +1,68 @@
+package ai.grakn.engine;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+/**
+ * <p>
+ * This class instantiates an engine server-side component once, statically until the JVM exists.
+ * Any state cleanup needs to be done by tests themselves - more code, more work, but more performant
+ * test suite. So all unit tests will share the same components and will be written with that in mind.  
+ * </p>
+ * 
+ * @author borislav
+ *
+ */
+public class EngineTestHelper {
```

Comment:
Yes, maybe. We can move it then. Or pull it out of the tests altogether and have it in the main artifact as a utility class to start an embedded engine. But to share code b/w module tests and the integration tests, the idea is to put that code in the test-tools module, right?  The question then is what are dependencies of this module? Is it ok for it to depend on any of the other grakn modules?